<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportSQLDatabaseWidget</class>
 <widget class="QWidget" name="ImportSQLDatabaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="lPassword">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="lePassword">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="sbStartRow">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="leHostName"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="lPort">
     <property name="text">
      <string>port</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <widget class="QWidget" name="wPreviewTable" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QPushButton" name="pbRefreshPreview">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTableWidget" name="twPreviewTable"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lPreviewTable">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>355</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pbConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lHostName">
     <property name="text">
      <string>Host Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QComboBox" name="cbDatabaseTree"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lVendor">
     <property name="text">
      <string>Vendor</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lDatabaseTree">
     <property name="text">
      <string>Database Model:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="sbPort">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QSpinBox" name="sbEndRow">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QPushButton" name="pbImport">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="cbShowPreview">
     <property name="text">
      <string>Show Preview</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lStartRow">
     <property name="text">
      <string>Start Row:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lUserName">
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="lEndRow">
     <property name="text">
      <string>End Row:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="leUserName"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbVendor"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="leDatabaseName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lDatabase">
     <property name="text">
      <string>Database Name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" rowspan="2" colspan="4">
    <widget class="QWidget" name="wProjectTree" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lImportSheet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Import Sheet:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
